tests/codex/test-codex-skills.sh

```
@@ -0,0 +1,121 @@
+
+# 8. Symlink creation dry run
+TMP_DIR=$(mktemp -d)
+trap 'rm -rf "$TMP_DIR"' EXIT
+
+for skill_dir in "$ROOT/skills"/*/; do
+  [ -d "$skill_dir" ] || continue
+  skill_name="metaswarm-$(basename "$skill_dir")"
+  ln -sf "$skill_dir" "$TMP_DIR/$skill_name"
+done
+
+linked=$(ls -1 "$TMP_DIR" | wc -l | tr -d ' ')
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)